### PR TITLE
🐈 Added a cute cat variant

### PR DESCRIPTION
**Pull request by Kalitsune**
ADDED:
- a new cat variant (-k2 / --kitty2)
- a new pair of eyes (confused / looking around eyes)

image

### CODE REVIEW

**Comment by cybardev** [comment]
Thanks for the contribution, @Kalitsune 😊 The new kitty is indeed very cute :3

I'll merge as soon as I can test it. A bit blocked by other tasks, but I'll get to it hopefully within a day or two.

File: cutefetch
Comment (cybardev) [inline cutefetch:79]: Maybe add a comment like "sysinfo with another cute kitty"?

If not, that's fine too. I can add it after merging. Thanks.

**Review by cybardev — CHANGES_REQUESTED** [review]
Code looks fine. Needs a comment for the function. Will merge afterwards.

File: cutefetch
Comment (Kalitsune) [reply cutefetch:79]: Sure! Will do that ASAP

**Review by cybardev — APPROVED** [review]
Looks good to me. Thanks

**Comment by cybardev** [comment]
Still yet to test. Will do this afternoon and merge.

**Review by cybardev — CHANGES_REQUESTED** [review]
Sorry for undoing the approval. I have tested it to work and am satisfied with the code, but there needs to be a little update in the documentation to reflect your wonderful additions.

If you could do one more change - to modify the README.md file and add the -k2 flag and eye 14 - then that would be amazing. Currently it is as follows:

image

For -k2 you can put the same as -k1 and then "(2nd variant)", as you did in the code comment.

Thanks a lot.

**Comment by Kalitsune** [comment]
Is this alright? thanks for the returns and I apologize for the troubles

**Review by cybardev — APPROVED** [review]
Looks good to me. Thanks for your contribution. 😊

**Comment by cybardev** [comment]
> Is this alright? thanks for the inputs and I apologize for the trouble

It's great. No trouble at all. Sorry for requesting so many changes.

Thank you very much for your valuable contribution. Merging now.

**Comment by Kalitsune** [comment]
Sure, my terminal fetch has never be so minimalist and cute :)

**Comment by cybardev** [comment]
This is closed now but I wanted to have your input on a different issue.

I notice that in your screenshot, the "heart" icon (beside Gnome Shell) is broken. I'm actually seeing something weird on my office ThinkPad with Pop!_OS too. But these days I mostly use a Mac, plus I'm not very familiar with fonts and icons.

So, my question was if you know a better WM icon or a heart icon that works on Linux?

image

This is what it looks like on my Mac, for reference.

**Comment by Kalitsune** [comment]
I don't know, it probably depends on the font that you're using,
I use [NerdFont](https://www.nerdfonts.com/) as my terminal font because it provides a lot of icons and is kinda the reference when it comes to icons in fonts.
I personally use Jetbrains Mono NF but I don't think the font family have a lot of impact on the icons as long as it comes from NerdFont. You should probably start looking by that. I'll make some research in the meantime.

**Comment by Kalitsune** [comment]
image
how about this glyph ?

**Comment by Kalitsune** [comment]
image
I find this one cuter though, 
probably because it's rounder.

let me know what you think of it, I'll make a pull request to bring the change to the main repo if you like either one of these

**Comment by cybardev** [comment]
I like the second one. The lines on the heart are thicker, so looks better imo.

Feel free to make a PR for the change. Thank you 😊

**Comment by Kalitsune** [comment]
sure np, doing that right away

**Pull request merged**
